samples/Aridka/Aridka.Client/Program.cs

```diff
@@ -49,13 +49,15 @@ public static async Task<string> GetTokenAsync(HttpClient client)
             var response = await client.SendAsync(request, HttpCompletionOption.ResponseContentRead);
             response.EnsureSuccessStatusCode();
 
-            var payload = JObject.Parse(await response.Content.ReadAsStringAsync());
-            if (payload["error"] != null)
+
+            using var document = JsonDocument.Parse(await response.Content.ReadAsStringAsync());
```

Comment:
Could we use `System.Net.Http.Json` instead of doing that? You'd get bonus point for deserializing the response to a `OpenIddictResponse` (part of the `OpenIddict.Abstractions` package) 😄